sqlalchemy-stubs/sql/functions.pyi

```diff
@@ -31,12 +31,13 @@ from .selectable import FromClause
 from .selectable import Select
 from .selectable import TableValuedAlias
 from .visitors import TraversibleType
+from ..orm.attributes import Mapped
```

Comment:
This is ORM, this can't be in the SQL package

Reply:
I know, but in 1.4 Mapped is not a ClauseElement. since this is a minor thing it can be any